bootstrap.js

```diff
@@ -154,6 +154,16 @@ function loadImports(){
   Cu.import("resource://conversations/modules/plugins/embeds.js");
 }
 
+// This obserer is notified when a new window is created and injects our code
+let windowObserver = {
+  observe: function (aSubject, aTopic, aData) {
+    if (aTopic == "domwindowopened") {
+      loadImports();
+      aSubject.QueryInterface(Ci.nsIDOMWindow);
+      monkeyPatchWindow(aSubject.window, true);
+    }
+  }
+}
```

Comment:
I think you can fix this in your next pull request, I don't think it
warrants a separate commit anyhow :)

~ jonathan

On 04/16/2015 11:26 PM, pianoslum wrote:
>
> In bootstrap.js
> <https://github.com/protz/GMail-Conversation-View/pull/949#discussion_r28572381>:
>
> > @@ -154,6 +154,16 @@ function loadImports(){
> >    Cu.import("resource://conversations/modules/plugins/embeds.js");
> >  }
> >  
> > +// This obserer is notified when a new window is created and injects our code
> > +let windowObserver = {
> > +  observe: function (aSubject, aTopic, aData) {
> > +    if (aTopic == "domwindowopened") {
> > +      loadImports();
> > +      aSubject.QueryInterface(Ci.nsIDOMWindow);
> > +      monkeyPatchWindow(aSubject.window, true);
> > +    }
> > +  }
> > +}
>
> And you merged anyway? Should I provide a patch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/protz/GMail-Conversation-View/pull/949/files#r28572381>.
>